configuration/cardano/mainnet-p2p-toplogy.json

```diff
@@ -0,0 +1,20 @@
+{
+  "LocalRoots": {
+    "groups": [
+      {
+        "localRoots": {
+          "addrs": [
+            {
+              "addr": "relays-new.cardano-mainnet.iohk.io",
+              "port": 3001
+            }
+          ],
+          "advertise": false
+        },
+        "valency": 1
+      }
+    ]
+  }
+  , "PublicRoots": []
+}
+
```

Comment:
This is wrong.
`relays-new.cardano-mainnet.iohk.io` is a public root peer.
```
{
  "LocalRoots": {
    "groups": [
      ]
  },
  "PublicRoots": [
    {
      "publicRoots": {
        "addrs": [
          {
            "addr": "relays-new.cardano-mainnet.iohk.io",
            "port": 3001
          }
        ],
        "advertise": false
      }
    }
  ]
}
```

Reply:
ok, I'll move it to `PublicRoots` then

Reply:
Done